autogpt_platform/backend/backend/data/workspace.py

```
@@ -0,0 +1,285 @@
+
+    Args:
+        workspace_id: The workspace ID
+        path_prefix: Optional path prefix to filter (e.g., "/documents/")
+        include_deleted: Whether to include soft-deleted files
+        limit: Maximum number of files to return
+        offset: Number of files to skip
+
+    Returns:
+        List of UserWorkspaceFile instances
+    """
+    where_clause: UserWorkspaceFileWhereInput = {"workspaceId": workspace_id}
```

Ruff (0.14.14)

Undefined name `UserWorkspaceFileWhereInput`

(F821)